.circleci/config.yml

```diff
@@ -88,10 +88,6 @@ jobs:
           name: debug | ruby version
           command: |
             ruby -v
-      - run:
-          name: debug | brew version
-          command: |
-            brew -v
```

Comment:
Since we started using orbs for shellcheck, normal testing doesn't need homebrew.